tests/testapp/migrations/0001_initial.py

```diff
@@ -327,7 +327,6 @@ class Migration(migrations.Migration):
                     models.ImageField(
```

Comment:
Issue:
My local build wants to remove this field, although, I'm not sure why as the field looks to exist still
https://github.com/wagtail/wagtail/blob/main/wagtail/images/models.py#L246


Would be great if this could be tested by someone else locally, to see if migration changes are needed and/or requested.

Reply:
Are you referring to the change below (line 330)? It's just removing the verbose name attribute, which makes sense as it matches the field name. If I run `makemigrations` against your branch I get the following output:

```
Migrations for 'testapp':
  testapp/migrations/0003_alter_blogpage_body_alter_customimage_file.py
    - Alter field body on blogpage
    - Alter field file on customimage
```

So it seems there's some changes that haven't been added to migrations. In the past, for projects with test apps in the past I've removed all the test app migrations and remade them fresh using the newest supported versions of Django and Wagtail - you just need to run tox on all the envs to make sure the migrations don't fail on any particular version.

Reply:
As discussed over slack, I've updated the migrations on django 4.2 and wagtail 5.2